packages/vuetify/src/components/VSlideGroup/VSlideGroup.ts

```diff
@@ -311,6 +328,10 @@ export const BaseSlideGroup = mixins<options &
           this.scrollOffset = maxScrollOffset
         }
       }
+
+      this.isSwiping = false
+      // rollback whole page scrolling to default
+      document.documentElement.style.removeProperty('overflow-y')
```

Comment:
Same here, need to check if IN_BROWSER

Reply:
I guess the whole `onTouchEnd` function should be guarded and checked by `typeof window !== 'undefined'` ? So `if (!this.canTouch) return ` is added to the begginning of this function in line 321